bin/serve

```diff
@@ -6,22 +6,35 @@ use SilverStripe\Core\Environment;
 
 define('BASE_URL', '');
 
+// Parse command-line options and env vars
+$options = getopt(null, [ 'host:', 'port:', 'hash:', 'bootstrap-file:', 'open', 'path:' ]);
```

Comment:
```suggestion
$options = getopt('', [ 'host:', 'port:', 'hash:', 'bootstrap-file:', 'open', 'path:' ]);
```
https://www.php.net/manual/en/function.getopt.php

Can't pull null to string params in php8.1, it'll result in a deprecation warning

Reply:
Updated